Encountify/Views/RegisterPage.xaml.cs

```diff
@@ -54,6 +56,10 @@ private async void OnRegisterClicked(object sender, EventArgs e)
                         user.Password = Password.Text;
                         user.Email = Email.Text;
                         RegisterUser(user);
+                        await DisplayAlert("Congratulations!", "You got an achievment, Welcome to Encountify!", "OK");
```

Comment:
This achievement message only for new users?

Reply:
Yes